examples/eventcounters/Examples.EventCounters.AspNetCore/Examples.EventCounters.AspNetCore.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
i'd suggest to not have examples here. The readme can explain the usage without a full blown example application.